crest/Assets/Crest/Crest/Scripts/Collision/CollProviderBakedFFT.cs

```diff
@@ -0,0 +1,421 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
```

Comment:
Adding the following to the top of the file (and for any other file using math) will soften the blow of not having the package installed:

```
#if !CREST_MATH
#error Crest requires the Unity Mathematics package (com.unity.mathematics) to be installed.
#endif
```

It will require a package version defines in the assembly definition.